### ISSUE TITLE
docker-compose down throws an error, if there has never been an up before

### ISSUE THREAD

**Issue by schmunk42** [docker/compose#3244]
Sample output for `docker-compose down` without running `up`

```
docker-compose -p "test-allow_failure" -f docker-compose.yml -f build/compose/test.override.yml down
Removing network testallowfailure_default
ERROR: network testallowfailure_default not found
```

IMHO a warning would be good enough here.

`docker-compose version 1.6.2, build 4d72027`

**Comment by aanand** [comment]
I believe #3135 fixes this - out in [1.7.0 RC1](https://github.com/docker/compose/releases/tag/1.7.0-rc1).

**Comment by dnephin** [comment]
Yup, this should be fixed. Going to close it, but please do comment or re-open if you can still reproduce on 1.7.0rc1
